src/azure_devtools/ci_tools/tests/Framework.py

```diff
@@ -0,0 +1,295 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do we need this? Can we import it through PIP?

Reply:
No we can't right now, since the file is patched. I made a PR but they didn't merge it yet.
I think it's fine to have this file, since it's not shipped at all, it's just local for testing. This will never end on PyPI part of devtools.
But still waiting a little if PyGithub can merge my PR, this won't be necessary